plugins/builds/create.js

```diff
@@ -26,8 +27,11 @@ module.exports = (server) => ({
                 tokenGen: (buildId) =>
                     request.server.plugins.login.generateToken(buildId, ['build'])
             };
+            const config = hoek.applyToDefaults(payload, {
+                username: request.auth.credentials.username
```

Comment:
This is great an all, but shouldn't we be checking if they can create the build first?  And if so, shouldn't we add them to the list?